jobbergate-api/jobbergate_api/apps/job_script_templates/services.py

```diff
@@ -1,14 +1,27 @@
 """Services for the job_script_templates resource, including module specific business logic."""
 
-from typing import Any
+from typing import Any, NamedTuple
 
 from buzz import require_condition
 from fastapi import status
-from sqlalchemy import not_
+from loguru import logger
+from pendulum.datetime import DateTime as PendulumDateTime
+from sqlalchemy import delete, func, not_, select, update
 from sqlalchemy.sql.expression import Select
 
 from jobbergate_api.apps.job_script_templates.models import JobScriptTemplate
+from jobbergate_api.apps.job_scripts.models import JobScript
 from jobbergate_api.apps.services import CrudModel, CrudService, FileService, ServiceError
+from jobbergate_api.config import settings
+
+
+class AutoCleanResponse(NamedTuple):
```

Comment:
I see this repeating a couple times, this could be moved to `apps/services` so it can be shared with all levels.

Reply:
That's true, I moved it.